new-lamassu-admin/src/pages/Wallet/helper.js

```diff
@@ -40,6 +49,12 @@ const getElements = (cryptoCurrencies, accounts, onChange, wizard = false) => {
     filterCoins(it)(filterOptions(option))
   )
 
+  const getZeroConfLimit = it => {
+    if (currentCurrency === 'Ethereum')
+      return <div style={classes.editDisabled}>{it}</div>
```

Comment:
I don't think we need the changes marked here if we just test it.ID against 'ETH' instead of using "currencCurrency"